scania-digital-design-system/sdds
# feat(data-table): mode variant prop and storybook examples update

**Describe pull-request**  
Adding a mode-variant feature to the data-table component, both native and web component

**Solving issue**  
Fixes: [AB#3115](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3115)

**How to test**  
1. Go to the Data table component in PR storybook link
2. Check the data-table component (both native and web component)
3. Make sure every data-table story has a "Mode-variant" control
4. Make sure every "Mode-variant" control works - it changes the background color of the data-table body row.
5. Compare colors for both dark and light mode and it variants: primary/secondary

**Suggested test steps**


**Additional context**  
1. Only body row is affected by mode-variant changes
2. For checking color, feel free to use my [Figma playground file](https://www.figma.com/file/hflfTzRXmy5dJQcm0EJOLm/Johnny-tests-table?node-id=0%3A1&t=q6mg5QLvTb8aaXty-1) with the basic data-table component.
